projects/packages/my-jetpack/_inc/components/product-cards-section/style.module.scss

```diff
@@ -47,12 +47,30 @@
 	}
 }
 
-.loading-placeholder {
-	margin-top: calc( var( --spacing-base ) / 2 ); // 4px
+.ownedProductsSection {
+	margin: -1.5rem;
+	padding: 1.5rem;
```

Comment:
This was a hacky way to get the background the full width of the page without having to adjust the margins on all the sections on the page. If the reviewer has a better suggestion I'm happy to change it 😄